doc/source/whatsnew/v0.18.1.rst

```diff
@@ -61,8 +61,8 @@ Tuesday after MLK Day (Monday is skipped because it's a holiday)
 
 .. _whatsnew_0181.deferred_ops:
 
-``.groupby(..)`` syntax with window and resample operations
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Method ``.GroupBy(..)`` syntax with window and resample operations
```

Comment:
```suggestion
Method ``.groupby(..)`` syntax with window and resample operations
```

If referring to the method should stick with `.groupby` The class can be capitalized

Reply:
Ok but we need to change the algorithm in validate_rst_title_capitalization, because at the moment, every word 'groupby' will be seen as an exception, and needed to be capitalised like 'GroupBy'.

Reply:
Can you leave these `.groupby` methods in lower case. Not sure what's the best solution here, it needs to be `GroupBy` when it's a class, and `groupby` when it's a method. And making the script be happy with both is not trivial, the way it is implemented.

So, let's leave them correct for now, even if we can't add the validation to this file, and see what we do with the script. If you can open an issue for this case, that would be great. You can tag me on the issue. Thanks!

Reply:
Ok, I will switch back to `.groupby` and I will open an issue! And try to propose a solution when I can spend some time on it.